doc/analytics.md

```diff
@@ -0,0 +1,75 @@
+# Analytics
+
+We use event and visit tracking to better recognize our user's needs. On the backend, we record our controllers and actions. On the client side, we track page's visited, elements clicked, and provide functions for JavaScript triggered events. Our primary tracking mechanism is based on the ```ahoy_matey``` gem. 
+
+For more information about ```Ahoy``` go to [the repo page](https://github.com/ankane/ahoy).
```

Comment:
just a markdown observation: inline verbatim markup only requires single backticks `like this` -- the triple ticks are only for block-style verbatim sections. Not a showstopper.